testing.sagebrush.master/Stepwat_Inputs/Input/sxw/Input/outsetup.in

```diff
@@ -62,8 +62,8 @@ TIMESTEP dy mo yr # must be lowercase
  INTERCEPTION     SUM       MO       1     end    interception      /* intercepted rain (cm): total, trees, shrubs, forbs, grasses, and litter (cm) */
      LYRDRAIN     SUM       DY       1     end     percolation      /* water percolated from each layer (cm) */
        HYDRED     SUM       WK       1     end          hydred      /* hydraulic redistribution from each layer (cm): total, trees, shrubs, forbs, grasses */
-          AET     SUM       YR       1     end             aet      /* actual evapotr. (cm) */
-          PET     SUM       DY       1     end             pet      /* potential evaptr (cm) */
+          AET     SUM       YR       1     end             aet      /* actual evapotranspiration (cm) */
+          PET     SUM       DY       1     end             pet      /* potential evapotranspiration (cm), extraterrestrial horizontal solar irradiation [MJ/m2], extraterrestrial tilted solar irradiation [MJ/m2], global horizontal irradiation [MJ/m2], global tilted irradiation [MJ/m2] */
```

Comment:
I can confirm these four new variables are present in the SOILWAT2 output within sw2_yearly_agg, sw2_monthly_agg, and sw2_daily_agg. 

The SD values for PET_H_oh_MJm-2 and PET_H_ot_MJm-2 where 0 across all iterations for each year (200 iterations, 300 years) in all files. Is this expected?

Reply:
Yes, this is expected: extraterrestrial radiation (H_oh and H_ot) are functions of latitude and day of year, i.e., they don't depend on meteorological and climatic conditions (which vary among years and iterations), whereas global horizontal and titled radiation (H_gh and H_gt) are affected by atmospheric conditions.